frontend/src/pages/projects/screens/detail/notebooks/NotebookSizeDetails.tsx

```diff
@@ -16,18 +16,28 @@ const NotebookSizeDetails: React.FC<NotebookSizeDetailsProps> = ({ notebookSize
     resources: { requests, limits },
   } = notebookSize;
 
+  const formatMemory = (memory: string | undefined): string => {
+    if (!memory) {
+      return 'Unknown';
+    }
+    if (/(Gi|Mi)$/.test(memory)) {
+      return `${memory}B`;
+    }
+    return memory;
+  };
```

Comment:
I suggest creating and making use of a reusable function in `valueUnits` which will convert any string in similar fashion to `splitValueUnit`:

```suggestion
const formatMemory = (value: ValueUnitString): ValueUnitString => {
  const match = value.match(/^(\d*\.?\d*)(.*)$/);
  if (!(match && match[1] && match[2])) {
    return value;
  }
  return `${match[1]}${MEMORY_UNITS_FOR_PARSING.find((o) => o.unit === match[2])?.name || match[2]}`;
}
```

Reply:
Thank you @christianvogt for this code suggestion - added `formatMemory` as a reusable function to `valueUnits`.